src/pre_process/m_check_patches.fpp

```diff
@@ -74,11 +76,15 @@ contains
                 elseif (patch_icpp(i)%geometry == 12) then
                     call s_check_ellipsoid_patch_geometry(i)
                 elseif (patch_icpp(i)%geometry == 13) then
-                    call s_check_3D_analytical_patch_geometry(i)
+                    call s_mpi_abort('geometry 13 (formerly "3D Hardcoded")'// &
+                                     'is no longer supported for patch '//trim(iStr)// &
+                                     '. Exiting.')
                 elseif (patch_icpp(i)%geometry == 14) then
                     call s_check_spherical_harmonic_patch_geometry(i)
                 elseif (patch_icpp(i)%geometry == 15) then
-                    call s_check_1d_analytical_patch_geometry(i)
+                    call s_mpi_abort('geometry 15 (formerly "1D Hardcoded")'// &
+                                     'is no longer supported for patch '//trim(iStr)// &
+                                     '. Exiting.')
                 elseif (patch_icpp(i)%geometry == 16) then
                     print *, '1d pressure sinusoid'
```

Comment:
i think we can remove all of these elseifs that just print something. i added them years ago and i don't think they are helping anyone.